packages/lib-classifier/src/store/utils/ClassificationQueue.js

```diff
@@ -77,12 +77,25 @@ class ClassificationQueue {
   async _saveClassification(classificationData, authorization) {
     let response
     try {
+      let tokenError
+      if (authorization) {
+        const token = authorization.replace('Bearer ', '')
+        const { data, error } = await auth.verify(token)
```

Comment:
The JWT payload data looks like:
```js
{ 
  id: 12345, // Panoptes user ID
  login: 'a-test-user', // Panoptes user login
  dname: 'Test User', // Panoptes user display name
  scopes: [], // authenticated permission scopes for this token
  admin: false // Panoptes user admin flag
}
```

See the Ruby client for details:
https://github.com/zooniverse/panoptes-client.rb/blob/bf0cffdf87b96bc97ed1924a09bb924d7c80d79d/lib/panoptes/client/authentication.rb

Reply:
We could maybe do something useful with this eg. double check that the token matches the stored user account, or add verification data to the classification metadata.

Reply:
Also, `getBearerToken` doesn't return a token, but rather an Authorization header. That was slightly confusing to figure out.